packages/experimental/Icon/src/Icon.tsx

```diff
@@ -69,16 +75,17 @@ function renderSvg(iconProps: IconProps) {
   // If a color for the icon is not supplied, fall back to white or black depending on appearance
   // Tracked by issue #728
   const iconColor = downgradeColor(color);
+  const accessible = iconProps.accessible ?? true;
```

Comment:
Seems cleaner to add `accessible: true` to `baseProps` below. The `mergeProps` would take care of overriding `accessible` if accessible was passed in as a prop.

Can be cleanup for later, though.

Reply:
Thanks for catching it. I missed that part